core/src/main/java/org/infinispan/expiration/impl/ExpirationManagerImpl.java

```diff
@@ -23,25 +23,29 @@
 
 import java.util.Iterator;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
 @ThreadSafe
 public class ExpirationManagerImpl<K, V> implements ExpirationManager<K, V> {
-   private static final Log log = LogFactory.getLog(ExpirationManagerImpl.class);
-   private static final boolean trace = log.isTraceEnabled();
-   ScheduledFuture <?> expirationTask;
+   protected final Log log = LogFactory.getLog(this.getClass());
+   protected final boolean trace = log.isTraceEnabled();
```

Comment:
This is a tad confusing, but probably still understandable since when a ClusterExpirationManager is in use that you will see ExpirationManagerImpl trace messages for some things.  But that should be okay.